.eslintrc

```diff
@@ -21,5 +21,6 @@
     "no-use-before-define": ["warn"],
     "react/no-multi-comp": ["warn"],
     "react/jsx-no-bind": ["warn"],
+    "react/forbid-prop-types": ["off"]
```

Comment:
Finding another solution is probably the most correct, but If you set it to `"warn"` instead of `"off"` it will still pass but just leave a warning message when linting